### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by damonkelley**
Accurate dev env setup.

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @somexpert @ldko

**Comment by somexpert** [comment]
:thumbsup:

File: README.md [hunk 1]
```diff
@@ -17,10 +17,18 @@ $ git clone https://github.com/unt-libraries/coda.git
 $ cd coda
 ```
 
+Warm up the MySQL database. This only needs to be done when the database container doesn't exist yet. This will take ~15 seconds once the image has been pulled.
+```sh
+$ docker-compose up -d mysql
```
Comment (ldko) [inline README.md]: Should this be `docker-compose up -d db`

**Comment by ldko** [comment]
Are you going to set up something with default secrets for the dev environment, or should we put into the README about setting up secrets.json?

**Comment by damonkelley** [comment]
Thanks for reminding me. I'll add a default secret to the dev environment.

**Comment by damonkelley** [comment]
@ldko, a local secret was added in 4d0ef38

**Comment by ldko** [comment]
:+1:

**Pull request merged**